### PR TITLE
Bug Catching Contest tweaks

### PR DESCRIPTION
**Pull request by CypherX**
## Description
A few tweaks to the bug catching contest to improve gameplay:
- Catching a shiny now awards x5 contest tokens
- A notification will now be shown displaying the amount of contest tokens earned upon catching a pokemon (lasts for 5 seconds)
- Added the ability to override the Safari Zone button text so it says `Bug Catching Contest` instead of `Enter Safari Zone`

I also wanted to add the players contest token amount to somewhere on the Safari modal but couldn't find a decent spot. Accepting ideas.

## How Has This Been Tested?
Notification appeared upon catching pokemon. Caught a shiny and verified 5x the normal amount of contest tokens was earned.

## Screenshots (optional):
image
image

### CODE REVIEW

File: src/scripts/towns/TownList.ts [hunk 1]
```diff
@@ -1934,7 +1934,7 @@ TownList['National Park'] = new Town(
     'National Park',
     GameConstants.Region.johto,
     GameConstants.JohtoSubRegions.Johto,
-    [new SafariTownContent(), JohtoContestShop],
+    [new SafariTownContent('Bug Catching Contest'), JohtoContestShop],
```
Comment (Qwertypop04) [inline src/scripts/towns/TownList.ts:1937]: It may be worth doing the same for the Friend Safari, as well?

Comment (CypherX) [reply src/scripts/towns/TownList.ts:1937]: Sure. Want it to say just 'Friend Safari' or 'Enter Friend Safari'?

Comment (Qwertypop04) [reply src/scripts/towns/TownList.ts:1937]: I guess "Enter Friend Safari" to match the Kanto Safari?

**Review by Qwertypop04 — APPROVED** [review]
Looks good, thanks for the PR!

**Pull request merged**